packages/viem-account-ledger/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@celo/viem-account-ledger",
   "version": "1.1.1",
   "description": "Helper library to make ledger<->viem interactions easier",
-  "type": "module",
```

Comment:
hmm this is a big change. no?

Reply:
yeah i dont like that, i just couldnt import it in the cli without